career_development/ladder.md

```diff
@@ -13,102 +13,112 @@
 	
-#### test-driven
--[ ] runs all tests before commiting, can understand where expectations are not met and fix some with the help of more senior developers
+#### Test-driven
+-[ ] Runs all tests before commiting.
+-[ ] Can understand where expectations are not met and fix some with the help of more senior developers.
 	
-#### iterative
--[ ] understands red-green-refactor
+#### Iterative
+-[ ] Understands red-green-refactor.
 	
-#### communication
--[ ] updates asana, comes prepared for standup, uses time tracking tools and vacation notifications
+#### Communication
+-[ ] Updates Asana.
+-[ ] Comes prepared for meetings.
+-[ ] Uses time tracking tools and vacation notifications.
 	
-#### respectful teammate
--[ ] on time for meetings, is enthusiastic
+#### Respectful teammate
+-[ ] On time for meetings.
+-[ ] Is enthusiastic.
 	
-#### hard skills
--[ ] can use git, linters, makes clear prs
+#### Hard skills
+-[ ] Can use git, linters, makes clear prs.
+-[ ] Can write code in at least one language.
 
 ### Level 2
-#### simple 	
-#### test-driven
--[ ] codes unit tests before code, writes acceptance tests	
-#### iterative
--[ ] delivers working code with each commit
-#### communication	
--[ ] hosts retros, can compose a PR and provide feedback on them, even small items
-#### respectful teammate
--[ ] shows respect for client goals, encourages team to stay on a sustainable pace
-#### hard skills
+#### Simple 	
+-[ ] Creates simple method / functions that most static code analysers would accept.
+#### Test-driven
+-[ ] Codes unit tests before code, writes acceptance tests.
+#### Iterative
+-[ ] Delivers working code with each commit.
+#### Communication	
+-[ ] Hosts retros.
+-[ ] Can compose a PR and provide feedback on them.
+#### Respectful teammate
+-[ ] Shows respect for client goals
+-[ ] Encourages team to stay on a sustainable pace.
+#### Hard skills
+-[ ] Understands what differentiates their favourite language with others. Knows what the language is useful for and when it is not the best option.
+-[ ] Understand where to place logic in their favourite framework.
 
 ### Level 3
-#### simple 	
-#### test-driven
--[ ] can start a project from scratch and setup testing frameworks; has an opinion about which frameworks to use; can mock http requests and understands where these tests will potentially fail	
-#### iterative	
--[ ] works with red-green-refactor, actively uses pomodoros,
-#### communication	
--[ ] is proficient with pair programming, can write user stories
-#### respectful teammate	
--[ ] actively participates in feedback for team members
-#### hard skills
+#### Simple 
+-[ ]
+#### Test-driven
+-[ ] Can start a project from scratch and setup testing frameworks.
+-[ ] Has an opinion about which frameworks to use.
+-[ ] Can mock http requests and understands where these tests will potentially fail.
+-[ ] Can integrate CI.	
+#### Iterative	
+-[ ] Works with red-green-refactor.
```

Comment:
Why is it important to "understands red-green-refactor" as a level 1 if it's not necessary to work with it until level 3?

Reply:
That's a good question. In level 1, the person nows the theoretical part of TDD, but they do not necessarily follow it. 
In level 3, the person has enough experience developing with this technique, so it's their default.